### PR TITLE
Q and A: updated formatting to nearly final state.

### PR DESCRIPTION
**Pull request by dylanreid7**
-Now uses proper Material UI components.
-Components split into separate files/folders.
-Questions loaded in order of "helpfulness," as suggested by the business requirements document.

### CODE REVIEW

File: client/components/QuestionsAndAnswers/QuestionsAndAnswers.jsx [hunk 1]
```diff
@@ -35,4 +28,19 @@ const QuestionsAndAnswers = () => {
   );
 };
 
+const sortByHelpfulness = (questionsAndAnswersData, numQuestions) => {
+  console.log(questionsAndAnswersData);
```
Comment (julia-thea) [inline client/components/QuestionsAndAnswers/QuestionsAndAnswers.jsx:32]: Some left behind logging statement here 😛

File: client/components/QuestionsAndAnswers/QAndAComponents/QAndA.jsx [hunk 2]
```diff
@@ -1,33 +1,68 @@
...
+            <Typography variant="caption">|</Typography>
+          </Grid>
+          <Grid item className={classes.textSpacing}>
+            <Typography variant="caption" className={classes.underlined}>Add Answer</Typography>
+          </Grid>
+        </Grid>
+      </Grid>
+      <Grid container spacing={2} xs={12}>
+        <Grid item>
+          {/* <Typography variant="h6"><b>A: </b>{props.question.answers[1628393].body}</Typography> */}
```
Comment (julia-thea) [inline client/components/QuestionsAndAnswers/QAndAComponents/QAndA.jsx:43]: Do you still need this line? Seems like a line no longer used and can be deleted.

**Review by julia-thea — COMMENTED** [review]
Your PR looks very clean. I appreciate the way you name your variables. It's very easy to read your code.
There was a logging statement left behind and a commented out line that looked like it could be deleted. Besides this I wanted to suggest to add some padding between the search bar and Q&As and between two buttons.
Otherwise looks really great! 🎉

File: client/components/QuestionsAndAnswers/QuestionsAndAnswers.jsx (hunk 1 above)
Comment (dylanreid7) [reply client/components/QuestionsAndAnswers/QuestionsAndAnswers.jsx]: Deleted, good catch!

File: client/components/QuestionsAndAnswers/QAndAComponents/QAndA.jsx (hunk 2 above)
Comment (dylanreid7) [reply client/components/QuestionsAndAnswers/QAndAComponents/QAndA.jsx]: I changed this to be a placeholder, which says "Future answer goes here."  Good catch, Julia :)

**Review by julia-thea — APPROVED** [review]

**Pull request merged**